apitools/base/py/batch_test.py

```diff
@@ -150,9 +150,18 @@ def testRequestServiceUnavailable(self):
                                   exceptions.HttpError)
 
     def testSingleRequestInBatch(self):
+        desired_url = 'https://www.example.com'
+
+        callback_was_called = []
```

Comment:
Python weirdness about callbacks and mutable variables. Calling a function on a variable will work 100% of the time, otherwise I need to mess with the global keyword.